Realm/RLMRealm.mm

```diff
@@ -276,7 +276,12 @@ + (void)setDefaultRealmPath:(NSString *)defaultRealmPath {
     s_defaultRealmPath = defaultRealmPath;
 }
 
-+ (NSString *)writeablePathForFile:(NSString*)fileName
++ (NSString *)writeableTemporaryPathForFile:(NSString *)fileName
+{
+    return [NSTemporaryDirectory() stringByAppendingPathComponent:fileName];
```

Comment:
I believe using `
- URLForDirectory:inDomain:appropriateForURL:create:error:` is recommended over `NSTemporaryDirectory`